admin/app/com/lucidchart/piezo/admin/controllers/Jobs.scala

```diff
@@ -150,4 +150,19 @@ class Jobs(schedulerFactory: WorkerSchedulerFactory) extends Controller {
 
     Ok(Json.obj("jobs" -> jobs.filter(_.getName.toLowerCase.contains(sofar.toLowerCase)).map(_.getName)))
   }
+
+  def runJob(group: String, name: String) = Action { implicit request =>
+    val jobKey = new JobKey(name, group)
+
+    if (scheduler.checkExists(jobKey)) {
+      try {
+        scheduler.triggerJob(jobKey)
+        Ok("")
+      } catch {
+        case e: SchedulerException => InternalServerError
```

Comment:
either log something, or let the exception propagate through to the default error handler, which will log something